### ISSUE TITLE
[Bug] "Could not do a head request" using Watchtower for automated Docker update

### ISSUE THREAD

**Issue by djkynch** [benphelps/homepage#1192]
### Description

Getting an error when using Watchtower:

```
level=warning msg="Could not do a head request for \"ghcr.io/benphelps/homepage:latest\", falling back to regular pull." container=/homepage image="ghcr.io/benphelps/homepage:latest"
level=warning msg="Reason: registry responded to head request with \"404 Not Found\", auth: \"not present\"" container=/homepage image="ghcr.io/benphelps/homepage:latest"
```

### Steps to reproduce

Added homepage docker container via docker-compose and watched with Watchtower.

### homepage version

v0.6.14 (a0fa636, Apr 5, 2023)

### Installation method

Docker

### Other

This was apparently discussed [here](https://github.com/containrrr/watchtower/discussions/1529#discussioncomment-4748053).

There is a resolution surrounding `docker buildx`.

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/en/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/benphelps/homepage/search?q=&type=issues) and [discussions](https://github.com/benphelps/homepage/search?q=&type=discussions).

**Comment by shamoon** [comment]
You can use the setting noted in that linked issue, the new manifest will be supported eventually. Also, this is a warning...

**Comment by kimpenhaus** [comment]
@djkynch that was fixed with release [1.5.2](https://github.com/containrrr/watchtower/releases/tag/v1.5.2)
